### PR TITLE
Added content for the emacs slides, changed date to that of the talk, and altered indentation.

### PR DESCRIPTION
**Pull request by haumea11**

### CODE REVIEW

File: presentation.tex [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 \title{Vim vs. Emacs}
 \author{Brandon Moore and John Markiewicz}
-\date{}
```
Comment (moore3071) [inline presentation.tex:10]: I switched it to use \date{} which should simply exclude date on the title page. I'm split on whether or not to include the date

File: presentation.tex [hunk 2]
```diff
@@ -16,76 +16,101 @@
...
+  \end{itemize}
 \end{frame}
 \note{List of resources}
 
 \end{document}
+
+%%% Local Variables:
+%%% mode: latex
+%%% TeX-master: t
+%%% End:
```
Comment (moore3071) [inline presentation.tex]: What's with these comments at the end? Did Emacs add these? ;)

**Review by moore3071 — COMMENTED** [review]
Get back to me on the date thing. And I guess I'll conform to using non-insane indentation.

**Comment by haumea11** [comment]
The pull request should be fixed now.

**Pull request merged**